lms/djangoapps/teams/static/teams/templates/edit-team-member.underscore

```diff
@@ -0,0 +1,14 @@
+<li class="team-member">
+    <a class="member-profile" href="<%= memberProfileUrl %>">
+        <img class="image-url" src="<%= imageUrl %>" alt="profile page" />
+    </a>
+    <div class="member-info-container">
+    	<span class="primary"><%= username %></span>
+    	<div class="secondary">
+    		<span id="date-joined"><%= dateJoined %></span>
+    		<span> | </span>
+    		<span id="last-active"><%= lastActive %></span>
+    	</div>
+    </div>
+    <a class="action-remove-member" data-username="<%= username %>" href=""><%- gettext("Remove") %></a>
```

Comment:
```href=""``` is an invalid value for an anchor element (so is #). If you are using an anchor just so you have something put an event listener on, you should probably be using a ```<button>``` not an anchor.
cc @clytwynec so we can catch this with a test

We also don't want many identical values for this button/anchor text, so can we append screen reader only text here that includes the username?
```<%- gettext("Remove") %><span class="sr">&nbsp;<%= username %></span>```